contribs/gnodev/pkg/dev/node_state_test.go

```diff
@@ -0,0 +1,189 @@
+package dev
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	emitter "github.com/gnolang/gno/contribs/gnodev/internal/mock"
+	"github.com/gnolang/gno/contribs/gnodev/pkg/events"
+	"github.com/gnolang/gno/gno.land/pkg/gnoclient"
+	"github.com/gnolang/gno/gno.land/pkg/gnoland"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNodeMovePreviousTX(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Not sure your context should be shared between tests in the suite -- maybe wrap this in a helper, and give it a reasonable timeout like 5s

Reply:
done efc8d31c12321e9ca6a5df1145597663ae35fa92